tests/helpers.py

```diff
@@ -779,12 +779,13 @@ def get_start_creation_time(interface, pvc_name):
     format = '%H:%M:%S.%f'
     # Get the correct provisioner pod based on the interface
     if interface == constants.CEPHBLOCKPOOL:
-        pod_name = pod.get_rbd_provisioner_pod().name
+        pod_name = pod.get_rbd_provisioner_pod()
```

Comment:
could you please combine get_rbd_provisioner_pod and get_cephfs_provisioner_pod to 'get_provisioner_pod' based on interface?

Reply:
Made this change as requested.